qtpy/__main__.py

```diff
@@ -0,0 +1,9 @@
+import qtpy.cli
```

Comment:
Should we add a copyright/license header here?

Reply:
We could, though I didn't because the content is just a snippit of standard boilerplate that is neither substantive nor original enough to actually be copyrightable in the first case. In any case, I added it along with a module-level docstring, and also fixed the no-longer-needed in Python 3 encoding line added to the other file.